picopass/scenes/picopass_scene_read_card_success.c

```diff
@@ -138,7 +138,7 @@ void picopass_scene_read_card_success_on_enter(void* context) {
 
         if(no_key) {
             furi_string_cat_printf(key_str, "No Key: used NR-MAC");
-        } else if(pacs->key) {
+        } else {
```

Comment:
@bettse I hope this one is correct )

Reply:
I had the same question and stared at it for a bit.  Then I realized that it's a byte array in a struct, so it can never be null so long as the struct exists, so the `if` was always true.